extra_data/validation.py

```diff
@@ -106,6 +121,7 @@ def check_indices(self):
                         first_shape=first.shape,
                         count_shape=count.shape,
                     )
+                    continue
```

Comment:
we return here because code will break later if `count` and `first` shape mismatch